### ISSUE TITLE
(CV) multiple prod VMs need unique yet similar configs

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#617]
Can we do this in single shared_config branch -- perhaps detect machine name and use that to indicate which storage roots, etc?

vs. having diff configs for each instance (16+ configs for prod ...)

Tony thinks there may be an example of this in existing shared_configs (not sure which branch)

See also #619

**Comment by julianmorley** [comment]
Define 'types' or 'roles' for PresCat instances in shared_configs?
- "I'm a PresCat DB! (there should only be one of me)"
- "I'm a PresCat CV box!"
- "I'm a PresCat R3 box!"
And then have the local code generate a unique ID based on that?

Then we could have roles like:
- Role Name: CV_storage_root_01
--  Task: CV
---- targets: storage_root_01

- Role: CV_storage_root_02
--  Task: CV
---- targets: storage_root_02

- Role: CV_roots_02_03
-- Task: CV
---- targets: storage_root_02, storage_root_03

- Role: Do_Many_Things
-- Task: CV
---- targets:  storage_root_02, storage_root_03
-- Task: MV
---- targets:  storage_root_04
-- Task: R3
---- targets:  storage_root_05

**Comment by tallenaz** [comment]
Instead of
```
moab:
  storage_roots:
    services-disk02: '/services-disk02'
```
How about something like
```
preservation_catalog_prod_a:
  moab:
    storage_roots:
      services-disk02: '/services-disk02'
```
And then
```
Settings.method(get_hostname).call.moab.storage_roots
```
???

**Comment by atz** [comment]
@tallenaz I think we resolved this via a different strategy, correct?

**Comment by tallenaz** [comment]
@atz I think we can consider this closed once #650 gets merged, since it more or less follows the strategy in the first comment here to use a "single shared_config branch -- perhaps detect machine name and use that to indicate which storage roots" . I've added a `Fixes` comment in that PR so that this will close automatically.
